Sources/SwiftSyntax/SyntaxData.swift

```diff
@@ -327,43 +331,52 @@ struct SyntaxData {
   ///   - index: The index pointing to where in the raw layout to place this
   ///            child.
   ///   - newChild: The raw syntax for the new child to replace.
+  ///   - newChildArena: The arena in which `newChild` resides.
   ///   - arena: The arena in which the new node will be allocated.
```

Comment:
All the names no longer match up 😅 `newChild` is actually `with` in terms of the API, and `newChildArena == rawNodeArena`. `arena == allocationArena`

Reply:
If you generate this comment using Xcode (Cmd-Option-/), it will use the internal parameter names instead of the external ones 🤷🏽